lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/RemoteConfiguration.java

```diff
@@ -84,15 +100,15 @@
   /**
    * The number of tries.
    */
-  @NamedParameter(doc = "The number of tries.", default_value = "3")
+  @NamedParameter(doc = "The number of tries.", default_value = "" + REMOTE_CONNECTION_NUMBER_OF_RETRIES)
   public static final class NumberOfTries implements Name<Integer> {
     // Intentionally empty    
   }
 
   /**
    * The timeout of connection retrying.
    */
-  @NamedParameter(doc = "The timeout of connection retrying.", default_value = "10000")
+  @NamedParameter(doc = "The timeout of connection retrying.", default_value = "" + REMOTE_CONNECTION_RETRY_TIMEOUT)
```

Comment:
If RemoteStage's timeout is less than the retrying timeout * `numOfRetries`, then the connection could be rejected while closing the RemoteStage. So I've fixed the timeout to guarantee `RemoteStageTimeout` > `numOfRetries` * `RetryingTimeout`. I've described the detail in [REEF-1753](https://issues.apache.org/jira/browse/REEF-1753). I will add a comment on this.  Thanks for pointing it out!